Source/Core/CommandLineOptions.cs

```diff
@@ -2118,6 +2126,8 @@ be n (default 500)
   /timeLimit:<num>
                 Limit the number of seconds spent trying to verify
                 each procedure
+  /rlimit:<num>
```

Comment:
This is far too vague. What does `rlimit` mean? What are its units?

I presume this corresponds to Z3's "rlimit". If so you should state this.